### ISSUE TITLE
Loss History Callbacks in Graph Model

### ISSUE THREAD

**Issue by anayebi** [fchollet/keras#523]
I have a Graph model which has one input and one output (the layers just aren't sequentially stacked). I know that according to the docs, Callbacks are intended to be passed to the .fit() method of the Sequential model. Now, I noticed that while the ModelCheckpoint Callback works fine with the Graph model, the LossHistory Callback to record loss history does not work (specifically, after training the Graph model, my history.losses is empty, where I set history = LossHistory()).

Is there a way to get the Loss History Callback to work with a Graph model (especially a Graph model with the same number of inputs and outputs as a Sequential model)?

**Comment by fchollet** [comment]
This is now fixed. 

The loss and validation loss displayed are the sum of the losses over all outputs (which is the optimization goal during training).
